spec/acceptance/nodesets/ubuntu-1804.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
We can generate nodesets on the fly in our acceptance tsets. No need to include the file.